pkg/controllers/podgroup/pg_controller_test.go

```diff
@@ -353,3 +353,313 @@ func TestAddPodGroup(t *testing.T) {
 		}
 	}
 }
+
```

Comment:
Can we also add a test case that create a exising pg, can should call update method of pg when sts is updated?

Reply:
I may have not enough time to do it. Maybe we can seek help from other members of the community. :)